nucypher/cli/characters/ursula.py

```diff
@@ -342,18 +335,24 @@ def ursula(click_config,
         """Paint an existing configuration to the console"""
 
         if not URSULA.federated_only:
+            blockchain = URSULA.staking_agent.blockchain
+
             emitter.echo("BLOCKCHAIN ----------\n")
             painting.paint_contract_status(emitter=emitter, blockchain=URSULA.blockchain)
-            current_block = URSULA.blockchain.w3.eth.blockNumber
+            current_block = blockchain.w3.eth.blockNumber
             emitter.echo(f'Block # {current_block}')
             # TODO: 1231
             emitter.echo(f'NU Balance (staker): {URSULA.token_balance}')
-            emitter.echo(f'ETH Balance (worker): {URSULA.blockchain.client.get_balance(URSULA.worker_address)}')
-            emitter.echo(f'Current Gas Price {URSULA.blockchain.client.gas_price}')
+            emitter.echo(f'ETH Balance (worker): {blockchain.client.get_balance(URSULA.worker_address)}')
+            emitter.echo(f'Current Gas Price {blockchain.client.gas_price}')
 
         emitter.echo("CONFIGURATION --------")
         response = UrsulaConfiguration._read_configuration_file(filepath=config_file or ursula_config.config_file_location)
-        return emitter.ipc(response=response, request_id=0, duration=0) # FIXME: #1216 - what are request_id and duration here?
+        return emitter.ipc(response=response, request_id=0, duration=0)  # FIXME: what are request_id and duration here?
+
+    elif action == "forget":
+        actions.forget(emitter, configuration=ursula_config)
+        return
```

Comment:
`forget` already exists in line 248, and that's a good place to have it since it doesn't need the Ursula, just the config.